model.go

```diff
@@ -108,8 +123,25 @@ func (m model) mergePullRequest(pr pullRequest) tea.Cmd {
 	}
 }
 
+func (m model) openInBrowser(pr pullRequest) tea.Cmd {
+	return func() tea.Msg {
+		_, err := gh.Run("pr", "view", "--web", pr.url)
+		return completedAction{
+			pr:     pr,
+			action: "opened",
+			err:    err,
+		}
+	}
+}
+
 type mergePullRequestMessage struct {
 	pr     pullRequest
 	result string
 	err    error
 }
+
+type completedAction struct {
```

Comment:
shouldn't this be called `openInBrowserMessage`?

Reply:
My thinking was that it could be used for other such actions if we add them in the future. But maybe we want to keep these messages "unique" to the command/action that generated them?

Reply:
Let's keep it simple like that for now, IMO!

Reply:
![](https://c.tenor.com/Voc2jy5W6fQAAAAC/roger-that.gif)